symplyphysics/laws/thermodynamics/pressure_from_the_concentration_of_molecules_and_kinetic_energy.py

```diff
@@ -19,6 +23,35 @@
 
 law = Eq(pressure, 2 / 3 * molecules_concentration * average_kinetic_energy)
 
+## Proof
+
+temperature = kinetic_energy.law.subs({
```

Comment:
This is equation, so it is better to name it like equation, eg `temperature_eq` to distinguish from symbols. Same for other equations.